src/app/components/create-edit-event-page/create-edit-event-page.component.scss

```diff
@@ -0,0 +1,86 @@
+@import '../../variables.scss';
+
+@mixin flex-container($flex-direction-value, $justify-content-value: center) {
+    display: flex;
+    flex-direction: $flex-direction-value;
+    justify-content: $justify-content-value;
```

Comment:
This is reusable code... you should add it to style.scss so it can be used throughout components